ibis/expr/types.py

```diff
@@ -611,6 +626,8 @@ def get_column(self, name):
         ref = ops.TableColumn(name, self)
         return ref.to_expr()
 
+    # should be renamed to column_names and columns should return
+    # with the TableColumn expressions
```

Comment:
Maybe? `columns` seems like an API that would be pretty egregious to break, since most people will probably do the following if `.columns` returned a sequence of `TableColumn`:

1. Run `.columns`
2. Realize that's not what they want
3. Run `[c.name for c in t.columns]`

Reply:
not sure if this comment should remain, i'm not sure what it conveys except that we should improve ibis :)

Reply:
bump